pyccel/stdlib/cuda_ndarrays/cuda_ndarrays.h

```diff
@@ -24,4 +24,13 @@ void _cuda_array_fill_double(double c, t_ndarray arr);
 t_ndarray   cuda_array_create(int32_t nd, int64_t *shape, enum e_types type, bool is_view, enum e_memory_locations location);
 int32_t         cuda_free_array(t_ndarray dump);
 int32_t         cuda_free_pointer(t_ndarray dump);
+
+__host__ __device__
+int32_t cuda_free_host(t_ndarray arr);
```

Comment:
I don't see a need to decorate `cuda_free_host` with `__host__` as there is not way for the `device` to call it on a `host` array or memory.

Reply:
Fixed b9cff43a4cd4ed5cb85c4a84cc28c9cf74d24c05

Reply:
I noticed that the commit doesn't seem to have the changes I mentioned.

Reply:
Yeah it was a mistake. Fixed 243ea719deb4621de9031638ff1e293a03faa280